attestation/context.go

```diff
@@ -138,16 +139,27 @@ func (ctx *AttestationContext) RunAttestors() error {
 	order := runTypeOrder()
 	for _, k := range order {
 		log.Debugf("Starting %s attestors...", k.String())
```

Comment:
Thinking of adjusting the log line here to make it clear that this is the start of a phase. And maybe add a Name method to OrderType so we can give a better visual feedback.